app/lib/pulp/v3/api.rb

```diff
@@ -48,6 +58,14 @@ def file_remotes_api
         @file_remotes_api ||= PulpFileClient::RemotesApi.new
       end
 
+      def file_content_api
+        @file_content_api ||= PulpFileClient::ContentApi.new
+      end
+
+      def artifacts_api
+        @file_content_api ||= PulpcoreClient::ArtifactsApi.new
```

Comment:
This looks like we are overwriting the definition of `@file_content_api` from line 62. Is that intentional?

Reply:
Blind Ctrl C & V :smile: I will rename it..